docs/pages/services/onto-docker.md

```diff
@@ -34,27 +33,27 @@ git clone https://git.material-digital.de/apps/ontodocker.git
 cd ontodocker
 ```
 
-### 2. Copy compose template
+### 2. Connect to SSO Identity Provider (IDP)
 
-```bash
-# copy (or link) the compose file template
-cp docker-compose-prod.yml docker-compose.yml
-```
+If you alreday have a valid client_secrets.json already, you must copy it to ```./data/oidc/```
+
+Otherwise in order to connect OntoDocker to the IDP you might need an initial access token (IAT) or generate one, if you want to connect it to your local Instance. (see "Initial Access Token" section of the [Keycloak manual](https://www.keycloak.org/docs/latest/securing_apps/#_initial_access_token))
 
-### 3. Connect to SSO Identity Provider (IDP)
-In order to connect OntoDocker to the IDP you might need an initial access token (IAT) or generate one, if you want to connect it to your local Instance. (see "Initial Access Token" section of the [Keycloak manual](https://www.keycloak.org/docs/latest/securing_apps/#_initial_access_token))
+Save the Initial Access Token as ```ia.jwt```
 
+Replace KEYCLOAK_URL, REALM_NAME in ```provider_info.json``` accordingly to the IDPs properties
+
+Set the APPLICATION_URL to the url where your instance is supposed to be accessible at
+
+Copy both ```ia.jwt``` and the customized ```provider_info.json``` files to ```./data/oidc/```
```

Comment:
Add some point we should probably try to have all configuration parameters in a single config.json and use this to automatically populate the custom config files

Reply:
Good idea